Documentation/gettingstarted/ipam-crd.rst

```diff
@@ -79,14 +80,19 @@ Create a CiliumNode CR
          [...]
        spec:
          ipam:
-           192.168.1.1: {}
-           192.168.1.2: {}
-           192.168.1.3: {}
-           192.168.1.4: {}
+           pool:
+             192.168.1.1: {}
+             192.168.1.2: {}
+             192.168.1.3: {}
+             192.168.1.4: {}
        status:
          ipam:
            used:
              192.168.1.1:
                owner: router
              192.168.1.3:
                owner: health
+
+Note
```

Comment:
It would be good to have it as a `reST` formatted note 

```
.. note::

    At the moment, only single IP addresses are allowed. CIDR's are not supported.
```